packages/components/src/templates/next/components/internal/Notification/Notification.tsx

```diff
@@ -14,19 +17,48 @@ const NotificationBanner = ({ content }: NotificationProps) => {
 
   return (
     isShown && (
-      <div className="flex flex-col justify-between gap-6 bg-site-primary-100 px-5 py-4 sm:flex-row sm:px-10">
-        <BaseParagraph
-          className="text-base text-content"
-          content={content}
-        ></BaseParagraph>
-        <button
-          className="ml-auto flex flex-row items-center gap-1"
-          aria-label="Dismiss notification"
-          onClick={onDismiss}
-        >
-          <p className="text-lg text-content">Dismiss</p>
-          <BiX className="h-full" />
-        </button>
+      <div className="bg-base-canvas-backdrop">
+        <div className="relative mx-auto flex max-w-screen-xl flex-row gap-4 px-6 py-8 text-base-content md:px-10 md:py-6">
+          <BiInfoCircle className="h-6 w-6 shrink-0" />
+          <div className="flex-1">
+            <Markdown
```

Comment:
issue: I think we should not be using Markdown here, since there are quite a few edge cases that we often have to debug from Classic days.

Instead, I think we should try to hook the Prose component up here, and have Studio render the Tiptap editor on the site settings page instead (since we are effectively using Tiptap's schema). That way, we are guaranteed that the rendering of the whole page, regardless of the component, is done by us and not offloaded to a third-party library.

Reply:
Oh ya talked to sehyun about this. Thought that this is purely an internal component (for us to banner); but realise can actually be used by admins. Going to move back to prose if admins can use it.